frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/emoji_popover.dart

```diff
@@ -50,14 +50,15 @@ class _EmojiPopoverState extends State<EmojiPopover> {
             ],
           );
         },
-        config: const Config(
+        config: Config(
           columns: 8,
           emojiSizeMax: 28,
           bgColor: Colors.transparent,
-          iconColor: Colors.grey,
-          iconColorSelected: Color(0xff333333),
-          indicatorColor: Color(0xff333333),
-          progressIndicatorColor: Color(0xff333333),
+          iconColor: Theme.of(context).iconTheme.color ?? Colors.blue,
```

Comment:
Let's use a `!` (null assertion) here because the purpose of calling the color from `Theme.of(context)` is to reduce code redundancy and the cost of refactoring. The theme isn't dynamically loaded so we will catch any null assertion errors during development.